src/theme/index.ts

```diff
@@ -35,7 +35,7 @@ export interface AppThemeType extends Omit<IdCheckThemeType, 'colors' | 'typogra
   isTabletViewport?: boolean // computed dynamically in ThemeProvider.tsx
   isDesktopViewport?: boolean // computed dynamically in ThemeProvider.tsx
   isTouch: boolean
-  showTabbar: boolean // computed dynamically in ThemeProvider.tsx
+  showTabbar?: boolean // computed dynamically in ThemeProvider.tsx
```

Comment:
Il y a une valeur par défaut

```suggestion
  showTabbar: boolean // computed dynamically in ThemeProvider.tsx
```